python/ray/_private/runtime_env/context.py

```diff
@@ -82,7 +82,8 @@ def exec_worker(self, passthrough_args: List[str], language: Language):
             )
         logger.debug(f"Exec'ing worker with command: {command_str}")
         if sys.platform == "win32":
-            subprocess.run([executable, *passthrough_args])
+            cmd = [*self.command_prefix, executable, *passthrough_args]
+            subprocess.Popen(cmd, shell=True).wait()
```

Comment:
There are some slight differences when running with `shell=True`. Is this intentional?

Reply:
I found that the tests ran more stable with `shell=True`. I honestly don't understand exactly why as the tests that fail with `shell=False` often pass when I run them individually. Maybe it has to do with the way pytest runs/keeps the state between runs, but I couldn't pin it down exactly.